src/prop-types.js

```diff
@@ -266,7 +266,6 @@ export const WalletType = PropTypes.shape({
   enable: PropTypes.bool.isRequired,
   connected: PropTypes.bool.isRequired,
   isContract: PropTypes.bool.isRequired,
-  networkType: PropTypes.string.isRequired,
   providerInfo: PropTypes.object.isRequired,
   web3: PropTypes.object.isRequired,
 })
```

Comment:
This type doesn't seem to be used anywhere, but is perhaps still useful for the sake of documentation?

Anyhow, I've updated it with our `wallet.js` changes.